matchms/filtering/add_compound_name.py

```diff
@@ -0,0 +1,23 @@
+from ..typing import SpectrumType
+
+
+def add_compound_name(spectrum_in: SpectrumType) -> SpectrumType:
+    """Add compound_name to correct field: "compound_name" in metadata."""
+    if spectrum_in is None:
+        return None
+
+    spectrum = spectrum_in.clone()
+
+    if spectrum.get("compound_name", None) is None:
+        # No compound_name entry found
```

Comment:
To me this comment is redundant as the code is already very clear. Removing the comment would make it cleaner in my opinion. But you can leave it if you like. Just a small thing.

Reply:
Done.